src/components/ImgEditor/ImgEditor.tsx

```diff
@@ -109,6 +113,13 @@ export default function ImgEditor<T extends FieldValues, K extends keyof T>(
       <p className="text-xs text-gray-d1 dark:text-gray mt-2">
         <span>
           Valid types are: PDF, JPG, PNG and WEBP. File should be less than 1MB.
```

Comment:
Remove PDF from acceptable file types. Slight tweaks to size wording. 
```suggestion
          Valid types are: JPG, JPEG, PNG and WEBP. Original uploaded image should be less than 1MB in size.
```